stable/openvpn/templates/openvpn-deployment.yaml

```diff
@@ -18,6 +18,8 @@ spec:
       labels:
         app: {{ template "openvpn.name" . }}
         release: {{ .Release.Name }}
+      annotations:
+        checksum/config: {{ include (print $.Template.BasePath "/config-openvpn.yaml") . | sha256sum }}
```

Comment:
The $ should not be needed here.

Reply:
Ok I will test it without, just copy/pasted from [Tips&Tricks](https://github.com/kubernetes/helm/blob/master/docs/charts_tips_and_tricks.md#automatically-roll-deployments-when-configmaps-or-secrets-change)